src/org/traccar/WebDataHandler.java

```diff
@@ -121,10 +129,50 @@ public String formatRequest(Position position) {
 
     @Override
     protected Position handlePosition(Position position) {
+        if (json) {
+            AsyncHttpClient.BoundRequestBuilder requestBuilder = Context.getAsyncHttpClient().preparePost(url);
+            requestBuilder.setBodyEncoding(StandardCharsets.UTF_8.name());
+
+            requestBuilder.addHeader("Content-Type", getContentType());
 
-        Context.getAsyncHttpClient().prepareGet(formatRequest(position)).execute();
+            setContent(position, requestBuilder);
+            requestBuilder.execute();
 
+        } else {
+            Context.getAsyncHttpClient().prepareGet(formatRequest(position)).execute();
+        }
         return position;
     }
 
+    protected String getContentType() {
```

Comment:
Why do we need a separate method?

Reply:
How would you want me to get the content type from 'JsonTypeEventForwarder' ??

Reply:
What does `JsonTypeEventForwarder` have to do with this? It has nothing to do with events.

Reply:
What did you mean by separate method ? maybe I dont get it.

Reply:
Why do you have a method with one line of code? It doesn't make sense to me.

Reply:
understood. Fixed and pushed